ui/views/components/head.ejs

```diff
@@ -0,0 +1,72 @@
+<meta charset="utf-8" />
+<meta http-equiv="X-XSS-Protection" content="1;mode=block" />
+<meta http-equiv="X-Content-Type-Options" content="nosniff" />
+<meta http-equiv="Permissions-Policy" content="interest-cohort=()" />
+<meta http-equiv="Referrer-Policy" content="strict-origin-when-cross-origin" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<meta name="theme-color" content="#235DF4" />
+<meta name="description" content="" />
+
+<title>DanaSwap App</title>
+
+<link rel="icon" href="./assets/images/favicon.ico" />
+
+<link rel="stylesheet" id="theme-css" href="./assets/styles/night.css" />
+
+<link
+  rel="stylesheet"
+  href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+/>
+
+<script>
+  document.addEventListener("DOMContentLoaded", () => {
+    // Get all "navbar-burger" elements
+    const $navbarBurgers = Array.prototype.slice.call(
+      document.querySelectorAll(".navbar-burger"),
+      0
+    );
+
+    // Add a click event on each of them
+    $navbarBurgers.forEach((el) => {
+      el.addEventListener("click", () => {
+        // Get the target from the "data-target" attribute
+        const target = el.dataset.target;
+        const $target = document.getElementById(target);
+
+        // Toggle the "is-active" class on both the "navbar-burger" and the "navbar-menu"
+        el.classList.toggle("is-active");
+        $target.classList.toggle("is-active");
+      });
+    });
+
+    // theme toggle
```

Comment:
There's a bit of a precedence that might be worth following here which would let some user agents change the style sheet without assistance from our JavaScript as well: https://developer.mozilla.org/en-US/docs/Web/CSS/Alternative_style_sheets

Reply:
ahh, good info, but does that mean the user has to `go to View>Page Style submenu and select a different style`?

Reply:
Oh, no. Chrome doesn't even have this option. I mean you can use this as a guide for the API and you could support both.